app/services/search/chat_channel_membership.rb

```diff
@@ -31,8 +31,8 @@ def set_query_size(params)
       end
 
       def paginate_hits(hits, params)
-        start = (params[:per_page] + 1) * params[:page]
-        hits[start, params[:per_page]]
+        start = params[:per_page] * params[:page]
+        hits[start, params[:per_page]] || []
```

Comment:
Previously, this line could return `nil` under certain conditions such as the following:

```ruby
hits = ["first_record", "second_record"]
params = { page: 1, per_page: 3 }

start = params[:per_page] * params[:page]
# => 3

hits[start, params[:per_page]]
# hits[3, 1]
# => nil
```

In plainer English...if a user were using a page size of 3 for a search that yielded 2 results total, and then requested the next page, you would get `nil` instead of `[]`. We always want to return `[]` to stay consistent.